src/common/types.ts

```diff
@@ -30,3 +31,18 @@ export interface customizedIconsPayload extends iconsPayload {
     exportSize?: number,
     customizationConfig?: customizedConfig
 }
+
+// *********************************
+// Themes:
+// *********************************
+
+// Available themes:
+export enum iconsTheme {
+    outlined,
+    svg
```

Comment:
both are svg, this can create confusion.
themes should be outlined and filled

Reply:
This would require a changes to the frontend, since frontend is using `svg-outlined`, `svg` am I right? 
Anyway, I need to update it to `svg-outlined` if this's the case

Reply:
I'm not really sure the names that the frontend is currently using, I will check the code, I couldn't find it in postman

Reply:
Found it, 
```JS 
app.get('/icon/svg/download/:iconName', function (req, res) {
  const theme = req.query.theme ? 'svg-outlined' : 'svg';
```

seems like we need to make it `svg-outlined` and `svg` to avoid making changes to `v1`, am I right? @kbhutani0001

Reply:
you can just keep the enum as filled, and the directory corresponding to it as the `svg/` one only. You are right, it should be changed accross everywhere as svg only doesnt make any sense. But we can update the enum atleast as it will be used everywhere across the BE

Reply:
I've added a type for `v1` to handle the case of `v1` theme's names, anyway, I'd prefer if we can change the FE, it just require changing one param.

Could you check? I couldn't find a better way to handle both cases.

Reply:
Now i get it what you meant, my bad. But having filled as enum makes more sense now

Reply:
Yeah, it will be used everywhere across the BE, you were totally right, it looks better now.